app/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Please add a newline.

Reply:
This is not a souce file, but rather a configuration file